### PR TITLE
Add com.snowplowanalytics.oss/oss_context/jsonschema/1-0-2

### PR DESCRIPTION
**Pull request by spenes**
https://github.com/snowplow/iglu-central/issues/1344

### CODE REVIEW

File: schemas/com.snowplowanalytics.oss/oss_context/jsonschema/1-0-2 [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    },
+    "appGeneratedId": {
+      "type": [ "string", "null" ],
+      "format": "uuid",
+      "description": "Automatically generated UUID on every application restart"
+    },
+    "installationId" : {
+      "type": [ "string", "null" ],
+      "maxLength": 256,
+      "description": "Unique id that identifies an installation"
```
Comment (stanch) [inline schemas/com.snowplowanalytics.oss/oss_context/jsonschema/1-0-2:65]:
```suggestion
      "description": "Unique id that identifies the installation, i.e. a set of pipelines running in the same infrastructure"
```

**Review by stanch — APPROVED** [review]

**Comment by spenes** [comment]
Hi @jbeemster, do we have any incoming Iglu Central release ? If we have, would it possible to add this schema to that release ? If we don't, I can prepare one.

**Comment by jbeemster** [comment]
Not as far as I am aware @spenes - please do start preparing a new release if you would like a new schema published!

**Comment by spenes** [comment]
@jbeemster Opened the [release PR](https://github.com/snowplow/iglu-central/pull/1346). Could you give it a look, please ? Closing this one.

**Pull request closed without merging**